packages/ethereum-contracts/scripts/deploy-test-token.js

```diff
@@ -16,7 +16,7 @@ const {
  * @param {Address} options.from Address to deploy contracts from
  * @param {boolean} options.resetToken Reset the token deployment
  *
- * Usage: npx truffle exec scripts/deploy-test-token.js : {TOKEN_SYMBOL}
+ * Usage: npx truffle exec scripts/deploy-test-token.js : {TOKEN_DECIMALS} {TOKEN_SYMBOL}
```

Comment:
okay...

Reply:
appending the arg and making it optional (keep default value) would have avoided breaking the interface